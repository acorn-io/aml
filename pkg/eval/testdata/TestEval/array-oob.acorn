foo: [1, 2]
bar: foo[2]