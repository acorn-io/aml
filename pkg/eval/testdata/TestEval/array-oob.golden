"index 2 out of bound, len 2: array-oob.acorn:2:6"
